### PR TITLE
#184948270 implementing add to cart 

### PR DESCRIPTION
**Pull request by wilbrord2**
#### What does this PR do?
This PR is for buyer/seller add product to cart
#### Description of Task to be completed?
- Implement the designs in react
- Integrate with the backend
- Jest testing 
#### How should this be manually tested?
- clone this repo `git clone https://github.com/atlp-rwanda/e-comm-team-amigos-fn`
- switch your working directory `cd e-comm-team-amigos-fn`
- create an `.env` and update it with `.env-example`
- install all dependencies by running `npm install`
- in your terminal run `npm run dev`
- route to endpoint `/cart`
#### Screenshots
Screenshot (191)
Screenshot (200)

### CODE REVIEW

**Review by GasanaJr — CHANGES_REQUESTED** [review]
The test files are failing; some data might have been lost during rebasing. you can check on develop and fix them

**Comment by Emile-SHUMBUSHO** [comment]
Align cart content at the center
Screenshot 2023-05-18 at 14 17 30

On successfully add to cart I'm suggesting to  redirect user to the cart  or somewhere else rather than staying on this UI otherwise this LGTM.
Screenshot 2023-05-18 at 14 18 34

**Review by GasanaJr — APPROVED** [review]
The functionality works perfectly fine. fix the unit tests and we are good to go

File: src/assets/css/Search.css [hunk 1]
```diff
@@ -1,73 +1,110 @@
...
-    margin: 0 auto 20px auto;
-  }
-  
\ No newline at end of file
+  flex-basis: 25%;
+  padding: 10px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  max-width: 300px;
+  margin: auto;
+  margin-top: 20px;
```
Comment (Habinezajanvier) [inline src/assets/css/Search.css]: Make sure that we are sticking to scss

**Review by Habinezajanvier — CHANGES_REQUESTED** [review]

File: src/assets/css/Search.css [hunk 2]
```diff
@@ -1,73 +1,110 @@
...
+}
+
+.card img {
+  margin-top: 15px;
+  margin-left: 40px;
+}
+
+.product-image {
+  width: 200px;
+  height: 200px;
```
Comment (Habinezajanvier) [inline src/assets/css/Search.css]: Make sure that we are sticking to scss not css

File: src/assets/css/addToCart.css [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+    margin: 0 auto;
+  }
+}
+.adcontainer .col-2 {
+  width: 40%;
+  padding: 1rem;
+}
+@media (max-width: 768px) {
+  .adcontainer .col-2 {
+    width: 90%;
```
Comment (Habinezajanvier) [inline src/assets/css/addToCart.css]: SCSS please not vanilla css

**Review by Habinezajanvier — CHANGES_REQUESTED** [review]

**Review by ericndungutse — APPROVED** [review]

**Review by ericndungutse — APPROVED** [review]

**Review by Mystica52 — APPROVED** [review]
LGTM unless rebase

**Review by Habinezajanvier — APPROVED** [review]
👍

**Pull request merged**